### PR TITLE
Add navigation grid documentation

### PR DESCRIPTION
**Pull request by goremikins**
## Description

Navigation grid documentation.

## Type of change

- [X] Documentation

### CODE REVIEW

**Comment by TremayneChrist** [comment]
Would be useful to have an example showing how this can be paired up with `tabindex` and arrow keys.

**Comment by goremikins** [comment]
@TremayneChrist , I would agree, but the sample is rather big. Maybe I can add it at the end of the document...

**Comment by goremikins** [comment]
@TremayneChrist , I have added a sample

**Comment by Nantawat-Poothong** [comment]
I think it would be great if we had sample code to show how to integrate this with our elements. Or may be it not necessary, what do you think?

**Comment by goremikins** [comment]
@Nantawat-Poothong , maybe. To make it "Our Element" we would need to wrap that code inside `BasicElement`.

File: documents/src/pages/utils/navigation-grid.md [hunk 1]
```diff
@@ -5,8 +5,443 @@ type: page
...
+
+## Types
+
+The list of _TypeScript_ types.
+
+| Name | Description | Value |
+| --- | --- | --- |
+| CellIndex | Cell index as `[Column Index, Row Index]`. _index_ is 0-based. | `[number, number]` |
+| NavigationGrid | A list of `NavigationRow` | `NavigationRow[]` |
+| NavigationRow | A list of _0_ and _1_. _0_ is an inactive cell, but _1_ is active. | `(0 &#124; 1)[]` |
```
Comment (Nantawat-Poothong) [inline documents/src/pages/utils/navigation-grid.md]:
```suggestion
| NavigationRow | A list of _0_ and _1_. _0_ is an inactive cell, but _1_ is active. | `(0 | 1)[]` |
```
I think we can use "|" directly, when it wraps with backticks.

Comment (goremikins) [reply documents/src/pages/utils/navigation-grid.md]: maybe, my IDE built-in editor does not like it

**Comment by goremikins** [comment]
@TremayneChrist , are you happy with this one?

**Review by TremayneChrist — APPROVED** [review]

**Pull request merged**